### PR TITLE
change Nebula Graph name

### PR DESCRIPTION
**Pull request by foesa-yang**

### CODE REVIEW

File: docs-2.0/15.contribution/how-to-contribute.md [hunk 1]
```diff
@@ -68,14 +68,14 @@ Nebula Graph 项目有很多[仓库](https://github.com/vesoft-inc)，以 [nebul
   # 正确的格式为：
   # origin    git@github.com:$(user)/nebula.git (fetch)
   # origin    git@github.com:$(user)/nebula.git (push)
-  # upstream  https://github.com/vesoft-inc/nebula (fetch)
+  # upstream  https://github.com/vesoft-inc/NebulaGraph (fetch)
```
Comment (whitewum) [inline docs-2.0/15.contribution/how-to-contribute.md:71]: 错了

**Review by whitewum — COMMENTED** [review]
a minor error

File: docs-2.0/7.data-security/1.authentication/4.ldap.md [hunk 2]
```diff
@@ -85,17 +85,17 @@ OpenLDAP 支持的认证方式有两种：简单绑定认证和搜索绑定认
...
+3. [重启 NebulaGraph 服务](../../4.deployment-and-installation/manage-service.md)，让新配置生效。
 
 4. 进行登录测试。
 
   ```bash
   $ ./nebula-console --addr 127.0.0.1 --port 9669 -u test2 -p passwdtest2
   2021/09/08 03:49:39 [INFO] connection pool is initialized successfully
 
-  Welcome to Nebula Graph!
+  Welcome to NebulaGraph!
```
Comment (abby-cyber) [inline docs-2.0/7.data-security/1.authentication/4.ldap.md:96]: not sure whether we need to change this one

File: docs-2.0/8.service-tuning/2.graph-modeling.md [hunk 3]
```diff
@@ -32,15 +32,15 @@
...
+而 NebulaGraph {{ nebula.release }} 是强 Schema 的（行存型）系统，这意味着业务数据模型中的部分是“不应该经常改变的”，例如属性 Schema 应该避免改变。类似于 MySQL 中 ALTER TABLE 是应该尽量避免的操作。
 
 而点及邻边可以非常低成本的增删，因此可以将业务模型中“经常改变的部分”建模成点或边（关系），而不是属性 Schema。
 
 例如，在一个业务模型中，人的属性是相对固定的，例如“年龄”，“性别”，“姓名”。而“通信好友”，“出入场所”，“交易账号”，“登录设备”等是相对容易改变的。前者适合建模为属性，后者适合建模为点或边。
 
 ### 自环
 
-Nebula Graph是强Schema类型系统，使用`ALTER TAG`的开销很大，而且也不支持List类型属性，当用户需要为点添加一些临时属性或者List类型的属性时，可以先创建包含所需属性的边类型，然后为点插入一条或多条指向自身的边。查询时只需要查询指向自己的边属性。如下图所示。
+NebulaGraph是强Schema类型系统，使用`ALTER TAG`的开销很大，而且也不支持List类型属性，当用户需要为点添加一些临时属性或者List类型的属性时，可以先创建包含所需属性的边类型，然后为点插入一条或多条指向自身的边。查询时只需要查询指向自己的边属性。如下图所示。
```
Comment (abby-cyber) [inline docs-2.0/8.service-tuning/2.graph-modeling.md:43]:
```suggestion
NebulaGraph 是强 Schema 类型系统，使用`ALTER TAG`的开销很大，而且也不支持List类型属性，当用户需要为点添加一些临时属性或者List类型的属性时，可以先创建包含所需属性的边类型，然后为点插入一条或多条指向自身的边。查询时只需要查询指向自己的边属性。如下图所示。
```

File: docs-2.0/graph-computing/nebula-analytics.md [hunk 4]
```diff
@@ -124,20 +124,20 @@ sudo rpm -i nebula-analytics-{{plato.release}}-centos.x86_64.rpm  --prefix /home
...
+      # NebulaGraph 的 graphd 服务地址。
       --graph_server_addrs=192.168.8.100:9669  
-      # Nebula Graph 的登录用户名。
+      # NebulaGraph 的登录用户名。
       --user=root  
-      # Nebula Graph 的登录密码。
-      --password=nebula  
-      # 写回 Nebula Graph 时采用的模式: insert 和 update。
+      # NebulaGraph 的登录密码。
+      --password=NebulaGraph
```
Comment (abby-cyber) [inline docs-2.0/graph-computing/nebula-analytics.md:133]: do not change this one

File: docs-2.0/nebula-cloud/nebula-cloud-on-azure/5.solution/5.3.role-and-authority-management.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
 # 管理角色权限
 
-Cloud 上 Solution 的角色与 Nebula Graph 内核的角色权限不相同，详情信息，参见 [Cloud Solution 角色身份说明](../4.user-role-description.md)。
+Cloud 上 Solution 的角色与 NebulaGraph 内核的角色权限不相同，详情信息，参见 [Cloud Solution 角色身份说明](../4.user-role-description.md)。
 
-下表显示了 Cloud 的角色权限与Nebula Graph内置角色权限的对应情况。
+下表显示了 Cloud 的角色权限与NebulaGraph内置角色权限的对应情况。
```
Comment (abby-cyber) [inline docs-2.0/nebula-cloud/nebula-cloud-on-azure/5.solution/5.3.role-and-authority-management.md:5]:
```suggestion
下表显示了 Cloud 的角色权限与 NebulaGraph 内置角色权限的对应情况。
```

File: docs-2.0/nebula-cloud/nebula-cloud-on-azure/5.solution/5.3.role-and-authority-management.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
...
 
-|Cloud上的角色权限|Nebula Graph内置角色权限|
+|Cloud上的角色权限|NebulaGraph内置角色权限|
```
Comment (abby-cyber) [inline docs-2.0/nebula-cloud/nebula-cloud-on-azure/5.solution/5.3.role-and-authority-management.md:7]:
```suggestion
|Cloud 上的角色权限|NebulaGraph 内置角色权限|
```

**Review by whitewum — APPROVED** [review]

File: docs-2.0/nebula-dashboard-ent/2.deploy-connect-dashboard-ent.md [hunk 7]
```diff
@@ -70,7 +70,7 @@
     host: 192.168.8.157 # 连接的 MySQL 数据库的 IP 地址。
     port: 3306 # 连接的 MySQL 数据库的端口号。
     username: root # 登陆 MySQL 的账户。
-    password: nebula # 登陆 MySQL 的密码。
+    password: NebulaGraph # 登陆 MySQL 的密码。
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard-ent/2.deploy-connect-dashboard-ent.md:73]: do not change this

File: docs-2.0/nebula-dashboard-ent/2.deploy-connect-dashboard-ent.md [hunk 8]
```diff
@@ -211,7 +211,7 @@ $ sudo /dashboard/scripts/dashboard.service restart all #重启 Dashboard 所有
   === "SQLite"
 
       ```bash
-      Nebula Dashboard Enterprise version need license, please enter the license file path(~/nebula.license): /home/vesoft/license/nebula.license
+      NebulaGraph Dashboard Enterprise version need license, please enter the license file path(~/nebula.license): /home/vesoft/license/nebula.license
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard-ent/2.deploy-connect-dashboard-ent.md:214]: do not change this one

File: docs-2.0/nebula-dashboard-ent/2.deploy-connect-dashboard-ent.md [hunk 9]
```diff
@@ -337,7 +337,7 @@ sudo rpm -e <package_name>
...
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard-ent/2.deploy-connect-dashboard-ent.md:340]: ditto

File: docs-2.0/nebula-exchange/use-exchange/ex-ug-export-from-nebula.md [hunk 10]
```diff
@@ -128,17 +128,17 @@ CentOS 7.9.2009
...
-    -rw-r--r--   3 nebula supergroup        172 2021-11-05 07:36 /vertex/player/    part-00003-17293020-ba2e-4243-b834-34495c0536b3-c000.csv
-    -rw-r--r--   3 nebula supergroup        144 2021-11-05 07:36 /vertex/player/    part-00004-17293020-ba2e-4243-b834-34495c0536b3-c000.csv
-    -rw-r--r--   3 nebula supergroup        173 2021-11-05 07:36 /vertex/player/    part-00005-17293020-ba2e-4243-b834-34495c0536b3-c000.csv
-    -rw-r--r--   3 nebula supergroup        160 2021-11-05 07:36 /vertex/player/    part-00006-17293020-ba2e-4243-b834-34495c0536b3-c000.csv
-    -rw-r--r--   3 nebula supergroup        148 2021-11-05 07:36 /vertex/player/    part-00007-17293020-ba2e-4243-b834-34495c0536b3-c000.csv
-    -rw-r--r--   3 nebula supergroup        125 2021-11-05 07:36 /vertex/player/    part-00008-17293020-ba2e-4243-b834-34495c0536b3-c000.csv
-    -rw-r--r--   3 nebula supergroup        119 2021-11-05 07:36 /vertex/player/    part-00009-17293020-ba2e-4243-b834-34495c0536b3-c000.csv
+    -rw-r--r--   3 NebulaGraph supergroup          0 2021-11-05 07:36 /vertex/player/_SUCCESS
```
Comment (abby-cyber) [inline docs-2.0/nebula-exchange/use-exchange/ex-ug-export-from-nebula.md:131]: we don't change this right?please check the following ones

File: docs-2.0/nebula-exchange/use-exchange/ex-ug-import-from-clickhouse.md [hunk 11]
```diff
@@ -91,7 +91,7 @@
   # Spark 相关配置
   spark: {
     app: {
-      name: Nebula Exchange {{exchange.release}}
+      name: NebulaGraph Exchange {{exchange.release}}
```
Comment (abby-cyber) [inline docs-2.0/nebula-exchange/use-exchange/ex-ug-import-from-clickhouse.md:94]: not sure we need to change this, please also change this same issue below.

File: docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md [hunk 12]
```diff
@@ -255,17 +255,17 @@
   返回：
 
   ```bash
-  nebulacluster.apps.nebula-graph.io/nebula created
+  nebulacluster.apps.nebula-graph.io/NebulaGraph created
```
Comment (abby-cyber) [inline docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md:258]: do not change this one

File: docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md [hunk 13]
```diff
@@ -288,22 +288,22 @@
 
   ```bash
   NAME     GRAPHD-DESIRED   GRAPHD-READY   METAD-DESIRED   METAD-READY   STORAGED-DESIRED   STORAGED-READY   AGE
-  nebula   1                1              1               1             3                  3                86s
+  NebulaGraph   1                1              1               1             3                  3                86s
```
Comment (abby-cyber) [inline docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md:291]: ditto

File: docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md [hunk 14]
```diff
@@ -24,18 +24,18 @@
 3. 配置 Helm 的环境变量。
    
   ```bash
-  export NEBULA_CLUSTER_NAME=nebula         # Nebula Graph 集群的名字。
-  export NEBULA_CLUSTER_NAMESPACE=nebula    # Nebula Graph 集群所处的命名空间的名字。
-  export STORAGE_CLASS_NAME=gp2             # Nebula Graph 集群的 StorageClass。
+  export NEBULA_CLUSTER_NAME=NebulaGraph         # NebulaGraph 集群的名字。
```
Comment (abby-cyber) [inline docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md:27]: ditto

File: docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md [hunk 15]
```diff
@@ -24,18 +24,18 @@
...
+  export NEBULA_CLUSTER_NAMESPACE=NebulaGraph    # NebulaGraph 集群所处的命名空间的名字。
```
Comment (abby-cyber) [inline docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md:28]: ditto

File: docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md [hunk 16]
```diff
@@ -105,23 +105,23 @@ helm uninstall "${NEBULA_CLUSTER_NAME}" --namespace="${NEBULA_CLUSTER_NAMESPACE}
 或者使用真实值删除集群，例如：
 
 ```bash
-helm uninstall nebula --namespace=nebula
+helm uninstall NebulaGraph --namespace=nebula
```
Comment (abby-cyber) [inline docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md:108]: ditto

File: docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md [hunk 17]
```diff
@@ -1,32 +1,32 @@
...
-本文示例中待升级的 Nebula Graph 版本为 {{operator.upgrade_from}}，其 YAML 文件名为`apps_v1alpha1_nebulacluster.yaml`。
+本文示例中待升级的 NebulaGraph 版本为 {{operator.upgrade_from}}，其 YAML 文件名为`apps_v1alpha1_nebulacluster.yaml`。
 
 ### 操作步骤
 
 1. 查看集群中服务的镜像版本。
 
   ```bash
-  kubectl get pods -l app.kubernetes.io/cluster=nebula  -o jsonpath="{.items[*].spec.containers[*].image}" |tr -s '[[:space:]]' '\n' |sort |uniq -c
+  kubectl get pods -l app.kubernetes.io/cluster=NebulaGraph  -o jsonpath="{.items[*].spec.containers[*].image}" |tr -s '[[:space:]]' '\n' |sort |uniq -c
```
Comment (abby-cyber) [inline docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md:29]: do not change this one

File: docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md [hunk 18]
```diff
@@ -124,7 +124,7 @@
 4. 等待约 2 分钟后，执行以下命令可查看到服务的镜像版本变更为 {{nebula.tag}}。
    
   ```bash
-  kubectl get pods -l app.kubernetes.io/cluster=nebula  -o jsonpath="{.items[*].spec.containers[*].image}" |tr -s '[[:space:]]' '\n' |sort |uniq -c
+  kubectl get pods -l app.kubernetes.io/cluster=NebulaGraph  -o jsonpath="{.items[*].spec.containers[*].image}" |tr -s '[[:space:]]' '\n' |sort |uniq -c
```
Comment (abby-cyber) [inline docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md:127]: ditto

File: docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md [hunk 19]
```diff
@@ -152,13 +152,13 @@
 2. 配置 Helm 的环境变量。
    
   ```bash
-  export NEBULA_CLUSTER_NAME=nebula         # Nebula Graph 集群的名字。
-  export NEBULA_CLUSTER_NAMESPACE=nebula    # Nebula Graph 集群所处的命名空间的名字。
+  export NEBULA_CLUSTER_NAME=NebulaGraph         # NebulaGraph 集群的名字。
```
Comment (abby-cyber) [inline docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md:155]: ditto

File: docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md [hunk 20]
```diff
@@ -152,13 +152,13 @@
...
+  export NEBULA_CLUSTER_NAMESPACE=NebulaGraph    # NebulaGraph 集群所处的命名空间的名字。
```
Comment (abby-cyber) [inline docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md:156]: ditto

File: docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md [hunk 21]
```diff
@@ -189,7 +189,7 @@
   查看集群版本：
 
   ```bash
-  $ kubectl get pods -l app.kubernetes.io/cluster=nebula  -o jsonpath="{.items[*].spec.containers[*].image}" |tr -s '[[:space:]]' '\n' |sort |uniq -c
+  $ kubectl get pods -l app.kubernetes.io/cluster=NebulaGraph  -o jsonpath="{.items[*].spec.containers[*].image}" |tr -s '[[:space:]]' '\n' |sort |uniq -c
```
Comment (abby-cyber) [inline docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md:192]: ditto

File: docs-2.0/nebula-studio/deploy-connect/st-ug-deploy.md [hunk 22]
```diff
@@ -65,7 +65,7 @@ $ sudo rpm -e nebula-graph-studio-{{studio.release}}.x86_64
 
 当屏幕返回以下信息时，表示 PRM 版 Studio 已经卸载。
 ```bash
-Nebula Studio removed, bye~
+NebulaGraph Studio removed, bye~
```
Comment (abby-cyber) [inline docs-2.0/nebula-studio/deploy-connect/st-ug-deploy.md:68]: not sure whether we need to change this one

File: docs-2.0/reuse/source_manage-service.md [hunk 23]
```diff
@@ -42,20 +42,20 @@ $ sudo /usr/local/nebula/scripts/nebula.service
...
+    - 对于使用企业版 Dashboard 安装的企业版 NebulaGraph，不支持使用`systemctl`管理服务。
     
-    - 对于其他方式安装的企业版 Nebula Graph，需要用户手动将`.service`文件移动到`/usr/lib/systemd/system`目录内，并修改`.service`文件内的`ExecStart`的文件路径，才可以正常使用`systemctl`命令。
+    - 对于其他方式安装的企业版 NebulaGraph，需要用户手动将`.service`文件移动到`/usr/lib/systemd/system`目录内，并修改`.service`文件内的`ExecStart`的文件路径，才可以正常使用`systemctl`命令。
     
 ### 语法
 
 ```bash
-$ systemctl <start | stop | restart | status > <nebula | nebula-metad | nebula-graphd | nebula-storaged>
+$ systemctl <start | stop | restart | status > <NebulaGraph | nebula-metad | nebula-graphd | nebula-storaged>
```
Comment (abby-cyber) [inline docs-2.0/reuse/source_manage-service.md:58]: do not change this one

**Pull request closed without merging**